.pre-commit-config.yaml

```diff
@@ -0,0 +1,7 @@
+repos:
+- repo: https://github.com/python-jsonschema/check-jsonschema
+  rev: 0.23.3
+  hooks:
+  - id: check-jsonschema
+    files: "^vulns/[a-z0-9_-]+/.+\\.yaml"
+    args: [--schemafile, "./.github/osv-schema.json"]
```

Comment:
@di if I'm understanding the implementation correctly, based on https://check-jsonschema.readthedocs.io/en/latest/usage.html#downloading-and-caching it can be invoked as 
`--schemafile https://raw.githubusercontent.com/ossf/osv-schema/main/validation/schema.json` or `--schemafile https://raw.githubusercontent.com/ossf/osv-schema/refs/tags/v1.6.7/validation/schema.json` if you wanted to explicitly pin to a version